bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/TeaserImpl.java

```diff
@@ -125,7 +125,14 @@ private void initModel() {
             if (!actions.isEmpty()) {
                 ListItem firstAction = actions.get(0);
                 if (firstAction != null) {
-                    targetPage = pageManager.getPage(firstAction.getPath());
+                	LOGGER.debug("firstaction URL : "+firstAction.getURL());
```

Comment:
I wouldn't assume that an action is only an internal page link if the path starts with `/content`.
I also wouldn't assume that it's external if it starts with `https`.